db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/analyze_deployment.rb

```diff
@@ -1,33 +1,22 @@
 def analyze_deployment
   $evm.log(:info, "********************** #{$evm.root['ae_state']} ***************************")
-  $evm.root['deployment_method'] = $evm.root['automation_task'].automation_request.options[:attrs][:deployment_method]
-
-  if $evm.root['deployment_method'] == "managed_provision"
+  if $evm.root['deployment_method'] == "provision"
     $evm.root['ae_next_state'] = "provision"
   else
     $evm.root['ae_next_state'] = "check_ssh"
-    $evm.root['deployment_master'] = $evm.root['automation_task'].automation_request.options[:attrs][:deployment_master]
     $evm.root['masters'] = $evm.root['automation_task'].automation_request.options[:attrs][:masters]
     $evm.root['nodes'] = $evm.root['automation_task'].automation_request.options[:attrs][:nodes]
+    $evm.root['deployment_master'] =  $evm.root['automation_task'].automation_request.options[:attrs][:deployment_master]
+    $evm.root['ssh_private_key'] = $evm.root['automation_task'].automation_request.options[:attrs][:ssh_private_key]
+    $evm.root['rhel_subscribe_inventory'] = $evm.root['automation_task'].automation_request.options[:attrs][:rhel_subscribe_inventory]
+    $evm.root['inventory'] = $evm.root['automation_task'].automation_request.options[:attrs][:inventory]
   end
-
-  $evm.root['deployment_type'] = $evm.root['automation_task'].automation_request.options[:attrs][:deployment_type]
-  $evm.root['user'] = $evm.root['automation_task'].automation_request.options[:attrs][:ssh_username]
-  $evm.root['password'] = $evm.root['automation_task'].automation_request.options[:attrs][:password]
-  $evm.root['provider_name'] = $evm.root['automation_task'].automation_request.options[:attrs][:provider_name]
-  $evm.root['private_key'] = $evm.root['automation_task'].automation_request.options[:attrs][:ssh_private_key]
-  $evm.root['manageiq_url'] = $evm.root['automation_task'].automation_request.options[:attrs][:manageiq_url]
-
-  # params needed for rhel subscriptions
-  $evm.root['rhsub_user'] = $evm.root['automation_task'].automation_request.options[:attrs][:rhsub_user]
-  $evm.root['rhsub_pass'] = $evm.root['automation_task'].automation_request.options[:attrs][:rhsub_pass]
-  $evm.root['rhsub_pool'] = $evm.root['automation_task'].automation_request.options[:attrs][:rhsub_pool]
-
-  # link the automation task to the container_deployment
+  $evm.root['ssh_username'] = $evm.root['automation_task'].automation_request.options[:attrs][:ssh_username]
   deployment = $evm.vmdb(:container_deployment).find(
```

Comment:
I see you're using $evm.root['container_deployment'] in other states as well so if you added it here, perhaps you can remove deployment (lines 15-16) and change 19 to:
`$evm.root['container_deployment'].add_automation_task($evm.root['automation_task'])`